### PR TITLE
add head of sales france job ad

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jacobpgn @jesssummerfield: please review.

### CODE REVIEW

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        </p>
+        <p className='para'>
+          At GoCardless we are looking to scale from processing $1 billion a year to over $10 billion.
+          Our vision is to create a new global payments network, making payments simpler on the internet
+          no matter what country you're in.
+          After entering France in early 2015,  we are rapidly gaining traction and now serve hundreds of
+          companies such as Habitat, La Fourchette and Zenchef.
+        </p>
+        <p className='para'>
+          We’re looking for a Head of Sales for France to drive our business in the region, who will lead from
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias change to "We're looking for a Head of Sales to drive our business in France..."

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          the front extending our current business and helping us learn and adapt to the requirements of the French market.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias change to "...lead from the front by extending our current business and by helping us to learn and..."

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          Reporting to the VP Sales, you will be the face of GoCardless in the French market and the advocate for
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias change to "Reporting to our VP Sales"

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+          France within our business. In addition to securing new key wins, you will be building out the enterprise
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias what is a new key win? Think we should be clearer here

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        </p>
+        <p className='para'>
+          You will coordinate with the Product team to develop new features for your market, and with the Marketing and
+          Partnership teams to generate a healthy pipeline of leads.
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>What we're looking for</h4>
+        <p className='para'>
+          We’re looking for people who want to have an impact and are always trying to learn and improve. You want to lead and
+          you feel ready to deal with the ambiguity present in small, fast growing environments. By evangelising GoCardless
+          and recurring payments, you are able to get customers excited by our product.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias this paragraph feels a bit blunt - can we soften it?

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+          <li>
+            You have experience closing complex SaaS deals with mid-level to CxO level.
+          </li>
+          <li>
+            You have a structured sales approach and experience with Salesforce.com.
+          </li>
+          <li>
+            You have great presentation and client management skills.
+          </li>
+        </ul>
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias I think there are too many bullet points here. Can we reduce to 5 key things? Would also be good to match other job ads, where we have a couple of sentences at the bottom on what an ideal candidate's background would be. I'm not clear from here what background would be a great fit for the position.

**Comment by gabrielamatias** [comment]
@jacobpgn : please review.

**Comment by jacobpgn** [comment]
👍

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+          companies such as Habitat, La Fourchette and Zenchef.
+        </p>
+        <p className='para'>
+          We’re looking for a Head of Sales to drive our business in France, who will lead from
+          the front by extending our current business and by helping us learn and adapt to the requirements of the French market.
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          Reporting to our VP Sales, you will be the face of GoCardless in the French market and the advocate for
+          France within our business. In addition to securing new key wins, you will be building out the enterprise
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias still unsure what a new key win means - can we clarify this please?

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+          attending key meetings. You will manage the development of Account Executives and Senior Account Executives by
+          conducting regular one on one meetings to identify growth opportunities and formalise training opportunities.
+        </p>
+        <p className='para'>
+          You will coordinate with the Product team to develop new features for your market, and with the Marketing and
+          Partnership teams to generate a healthy pipeline of leads.
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>What we're looking for</h4>
+        <p className='para'>
+          We’re looking for people who want to have an impact and are always trying to learn and improve. You have
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias second sentence still sounds a little blunt and also the experience leading teams/managing is duplicated in line 71. How about "We're looking for people who are able to get customers excited by our product, by evangelising GoCardless and recurring payments. We'd love to hear from you if want to have a lasting impact and are always trying to learn and improve."?

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+        </p>
+        <ul className='list u-margin-Tm'>
+          <li>
+            You have extensive B2B SaaS sales experience as an individual contributor.
+          </li>
+          <li>
+            You have built and managed a Sales team.
+          </li>
+          <li>
+            You have experience defining strategic sales plan and targets.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias should this be sales plans not plan?

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+          </li>
+          <li>
+            You have experience closing complex SaaS deals with mid-level to CxO level.
+          </li>
+          <li>
+            You have a structured sales approach and experience with Salesforce.com.
+          </li>
+        </ul>
+        <p className='para'>
+          This role would ideally suit someone with experience is a fast-growing SaaS business, with previous manager experience.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias should be "in" not "is" and might be worth losing the bit about previous manager experience as it's already in line 55

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+          However our team come from a variety of backgrounds and we welcome diversity – if you're unsure, please apply.
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias should be ndash!

File: app/pages/about/jobs/positions/head-of-sales-france.js (hunk 7 above)
Comment (gabrielamatias) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: It's a sales term. I can change to key deals.

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+      <JobsPage>
+        <h3 className='u-text-heading-light u-text-m u-color-heading'>Head of Sales: France</h3>
+        <p className='para'>
+          Grow the GoCardless business in France.
+        </p>
+        <p className='para'>
+          At GoCardless we are looking to scale from processing $1 billion a year to over $10 billion.
+          Our vision is to create a new global payments network, making payments simpler on the internet
+          no matter what country you're in.
+          After entering France in early 2015,  we are rapidly gaining traction and now serve hundreds of
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias is there a double space after the comma?

File: app/pages/about/jobs/positions/head-of-sales-france.js [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+          companies such as Habitat, La Fourchette and Zenchef.
+        </p>
+        <p className='para'>
+          We’re looking for a Head of Sales to drive our business in France, who will lead from
+          the front by extending our current business and by helping us learn and adapt to the requirements of the French market.
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          Reporting to our VP Sales, you will be the face of GoCardless in the French market and the advocate for
+          France within our business. In addition to securing new deals, you will be building out the enterprise
```
Comment (jesssummerfield) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: @gabrielamatias I know what key wins are! I mean I think we should clarify what that means for this role i.e. will they be only looking at larger customers

Comment (gabrielamatias) [inline app/pages/about/jobs/positions/head-of-sales-france.js]: ah I thought it was a more of "is this internal talk" vs "everyone in sales knows what they mean"

**Pull request merged**